test/test.js

```diff
@@ -9,6 +9,12 @@ const BinBuild = require('bin-build');
 const compareSize = require('compare-size');
 
 test.cb('rebuild the zopflipng binaries', t => {
+	// Skip the test on Windows
```

Comment:
Maybe there's a better way to skip the test on Windows? I'm not familiar with ava @sindresorhus

Reply:
Yes, this is the best way.